modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/DisasterRecoveryManager.java

```diff
@@ -556,35 +551,19 @@ private void handleLocalPartitionStatesRequest(LocalPartitionStatesRequest reque
 
                     CatalogTableDescriptor tableDescriptor = catalogManager.table(tablePartitionId.tableId(), catalogVersion);
                     // Only tables that belong to a specific catalog version will be returned.
-                    if (tableDescriptor == null || !containsOrEmpty(tableDescriptor.zoneId(), request.zoneIds())
-                    ) {
+                    if (tableDescriptor == null || !containsOrEmpty(tableDescriptor.zoneId(), request.zoneIds())) {
                         return;
                     }
 
-                    Node raftNode = raftGroupService.getRaftNode();
-                    State nodeState = raftNode.getNodeState();
-
-                    LocalPartitionStateEnum localState = convertState(nodeState);
-                    long lastLogIndex = raftNode.lastLogIndex();
-
-                    if (localState == HEALTHY) {
-                        // Node without log didn't process anything yet, it's not really "healthy" before it accepts leader's configuration.
-                        if (lastLogIndex == 0) {
-                            localState = INITIALIZING;
-                        }
-
-                        if (raftNode.isInstallingSnapshot()) {
-                            localState = INSTALLING_SNAPSHOT;
-                        }
-                    }
+                    LocalPartitionStateEnumWithLogIndex localPartitionStateWithLogIndex = of(raftGroupService.getRaftNode());
```

Comment:
I'm not a fan of static imports for methods called `of`, or `getInstance`, or something else non-descriptive. Please chose another name or don't use static import in this particular case, thank you

Reply:
I was wondering which name would be more appropriate, `convert` doesn’t seem to fit, maybe `toLocalPartitionStateWithLogIndex` or `getLocalPartitionStateWithLogIndex`, what do you think?

Reply:
I think we can avoid using static import here and that would be fine. We don't use it for `List.of` and `Set.of`, and nobody is upset about it, right?

Reply:
Reasonable, I agree.

Reply:
fix it